opencti-platform/opencti-front/src/private/components/analyses/opinions/StixCoreObjectOpinions.tsx

```diff
@@ -44,27 +77,57 @@ StixCoreObjectOpinionsProps
     () => fetchLoadQuery(variables, { fetchPolicy: 'store-and-network' }),
     [],
   );
+
+  const height = 260;
```

Comment:
Why just didn't define it directly in the style line ?

Reply:
Because it's used in different places

```
<StixCoreObjectOpinionsRadar
            queryRef={queryRef}
            height={height}
            opinionOptions={opinionOptions}
          />
```